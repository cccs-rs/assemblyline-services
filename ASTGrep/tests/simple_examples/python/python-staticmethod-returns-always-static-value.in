class lI1lI1l1lI1l1lll1II:
    @staticmethod
    def lIIlIl11(llIllllI):
        return True

lI1lI1l1lI1l1lll1II.lIIlIl11(123)
