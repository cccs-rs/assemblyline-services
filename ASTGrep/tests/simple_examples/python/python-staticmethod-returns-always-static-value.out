class lI1lI1l1lI1l1lll1II:
    @staticmethod
    def lIIlIl11(llIllllI):
        return True

True
